base/broadcast.jl

```diff
@@ -713,6 +713,32 @@ Like [`broadcast`](@ref), but store the result of
 Note that `dest` is only used to store the result, and does not supply
 arguments to `f` unless it is also listed in the `As`,
 as in `broadcast!(f, A, A, B)` to perform `A[:] = broadcast(f, A, B)`.
+
+# Examples
+```jldoctest
+julia> A = [1.0; 0.0];
+
+julia> B = [0.0; 0.0];
```

Comment:
Put `A` and `B` on the same line?

I noticed more and more that is becomes kinda annoying when the examples take too much vertical space so I try to condense as much as possible.